consensus/enclave/impl/src/lib.rs

```diff
@@ -401,7 +401,15 @@ impl SgxConsensusEnclave {
             }
 
             // The associated MintConfigTx should be valid.
-            self.validate_mint_config_txs(vec![mint_config_tx], current_block_index, config)?;
+            // Important note about the tombstone block: Since we are validating a
+            // MintConfigTx that has been previously added to the
+            // ledger, we shouldn't take the tombstone block into account. If we use
+            // current_block_index then a past MintConfigTx will eventually
+            // result in TombstoneBlockExceeded after enough blocks have been appended
+            // following it. The workaround here is to pretend that the current
+            // block index it is being validated against is within the tombstone range.
+            let config_block_index = mint_config_tx.prefix.tombstone_block.saturating_sub(1);
```

Comment:
Thoughts on updating `validate_mint_config_txs` to take an `Option<>` and if it's None then no block index validation is done?
It seems like it would: 
* reduce the comment and thus improve maintainability.  
* Remove the exposure of _how_ current_block_index is validated down lower.

The resultant comment might be
```rust
/// Don't validate `current_block_index` since this will only be called if
/// the MintConfigTx was previously added to the ledger
```

Reply:
I like it. Will do.